neo/Network/P2P/Payloads/PingPayload.cs

```diff
@@ -27,7 +27,7 @@ public static PingPayload Create(uint height, uint nonce)
             return new PingPayload
             {
                 LastBlockIndex = height,
-                Timestamp = DateTime.UtcNow.ToTimestamp(),
+                Timestamp = DateTime.UtcNow.ToTimestampMS(),
```

Comment:
Maybe we can leave the Ping in seconds.

Reply:
It is better to leave everything using the same "unit", what do you think?